zip-1014.rst

```diff
@@ -133,7 +143,7 @@ Starting at the first Zcash halving in 2020, until the second halving in 2024,
 20% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
 consists of the following three slices:
 
-* 35% for the Electric Coin Company (denoted **ECC slice**);
+* 35% for the Bootstrap Project (denoted **BP slice**);
```

Comment:
This raises the question of what this needs to be renamed to "BP slice" also in ZIP 214. I think it does, for consistency.

Reply:
Filed #412 (and https://github.com/zcash/zcash/pull/4830 for zcashd).

Reply:
https://github.com/zcash/zcash/pull/4830 is included in zcashd 4.1.0. The identifier for the BP slice is FS_ZIP214_BP. (This is just cosmetic, there is no consensus rule change other than the name and identifier of this stream.)